pmm/pmm2-server-autobuild.groovy

```diff
@@ -121,12 +126,14 @@ pipeline {
 
                             # 3rd-party
                             build-server-rpm clickhouse
-                            build-server-rpm consul
-                            build-server-rpm orchestrator
-                            build-server-rpm rds_exporter
+                            ### missing rpm spec in PMM-2.0
+                            # build-server-rpm consul
+                            # build-server-rpm orchestrator
+                            # build-server-rpm rds_exporter
```

Comment:
why?

Reply:
Do we have a separate RPM for rds_exporter?

Reply:
In PMM2 we don't have it